src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,18 +32,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        name="query"
+        value={query}
```

Comment:
The input field does not have a label, which can lead to accessibility issues. Consider adding a visually hidden label.